cfpb/sbl-frontend
# feat(email): Form Submission - Additional Three headers implementation

closes #244 

## Changes
- feat(email api): Update three headers and email subtitle titles based on [backend implementation](https://github.com/cfpb/regtech-mail-api/blob/email-api-take-1/regtech_mail_api/api.py#L49-51)
- feat(email api): Headers
- [x] subject = headers["X-Mail-Subject"]
- [x]  sender_addr = headers["X-Mail-Sender-Address"]
- [x]  (optional) sender_name = headers["X-Mail-Sender-Name"]
- feat(email api): Subject titles
- [x]  [BETA] Update your financial institution profile
- [x]  [BETA] Complete your user profile

## How to Test
- Code Inspect `submitUpdateFinancialProfile.ts` and `submitUserProfileFi.ts` to verify implementation
- Navigate to `/update-financial-profile`
- Fill out the form and hit 'Submit'
- In the *Network* Tab, you should see similar to the screenshot below

# Screenshot
<img width="1043" alt="Screenshot 2024-02-08 at 3 10 01 PM" src="https://github.com/cfpb/sbl-frontend/assets/13324863/35db0c11-06c1-457d-b8e1-f613d3c9b7ca">

## References
- Follow-up to https://github.com/cfpb/sbl-frontend/issues/239
- CURL Example - https://github.com/cfpb/regtech-mail-api/pull/4/files
